ReportPortal.Tests/ReportPortal.E2E.Core/Extensions/WebDriverExtensions.cs

```diff
@@ -47,5 +43,26 @@ public static bool ElementExistsByXPath(this IWebDriver driver, string xPath)
             var func = $"return document.evaluate(\"{xPath}\", document, null, XPathResult.BOOLEAN_TYPE, null).booleanValue";
             return driver.ExecuteJavaScript<bool>(func);
         }
+
+        public static void UpdateLambdaTestStatus(this IWebDriver driver, TestStatus status)
+        {
+            try
+            {
+                driver.ExecuteJavaScript("lambda-status=" + (status == TestStatus.Passed ? "passed" : "failed"));
+            }
+            catch
+            {
+                // ignored
+            }
```

Comment:
Why is it ignored?

Reply:
Ignored was added because According to my code style, the catch block cannot be empty. But I agree that adding a comment is not the best idea, the best solution would be to add log information. Added